packages/aws-cdk-lib/aws-lambda/lib/function.ts

```diff
@@ -625,6 +652,13 @@ export interface FunctionProps extends FunctionOptions {
    * the handler.
    */
   readonly handler: string;
+
+  /**
+   * The function's [AWS X-Ray](https://docs.aws.amazon.com/lambda/latest/dg/services-xray.html) tracing configuration. To sample and record incoming requests, set Mode to Active
+   *
+   * @see https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-lambda-function-tracingconfig.html#cfn-lambda-function-tracingconfig-mode
+   */
+  readonly tracingConfig?: TracingConfig;
```

Comment:
The `tracingConfig` property of L1 Construct has the following type

```ts
  export interface TracingConfigProperty {
    readonly mode?: string;
  }

public tracingConfig?: cdk.IResolvable | CfnFunction.TracingConfigProperty;
```

However, the type of the `mode` property of this `CfnFunction.TracingConfigProperty` is not strict, so L2 Construct does not use `CfnFunction.TracingConfigProperty`.

Instead, I have a more strict type (a type that uses enum for the `mode` property) and specify it.